src/screens/atomic-swap/ProposalList.js

```diff
@@ -63,7 +67,9 @@ export default function ProposalList (props) {
         for (const [proposalId, proposal] of Object.entries(proposals)) {
             const pId = proposalId;
             const password = proposal.password;
-            const pAmountTokens = proposal.data?.amountTokens; // Get this from proposal
+            const pAmountTokens = proposal.data
```

Comment:
Why this is a string? And why don't you use lodash `get` to handle the default values?

Reply:
If the proposal does not have a `data` property it means it is still being fetched from the backend, so the _amount of tokens_ column shows nothing.

If the `data` property is available we can decide to show the amount of tokens. The `'0'` placeholder is for the remote possibility that the current state of this proposal is between the `proposalFetchSuccess` and the `proposalUpdated` with its calculated metadata.

It seemed unnecessary to use `get` in this circumstance. Do you agree?

Reply:
If this value was kept as a number, there would be no way to differentiate a `0` as in "zero tokens for this proposal" from an `undefined` as in that there is no proposal data in memory.

The cleanest way to do this in code was to convert it to an empty string _or_ zero, when necessary.

Added an explicit conversion to string on [a8a7f29](https://github.com/HathorNetwork/hathor-wallet/pull/400/commits/a8a7f299285c617a572374c0b33734a1ec540f36) to make this clearer.

Reply:
I don't understand why there would be no way to differentiate `0` from `undefined`. They are clearly different and easy to differentiate in the code, isn't it?